tasks/main.yml

```
@@ -0,0 +1,46 @@
+---
+- name: Calculate facts
+  set_fact:
+    build_base: "{{ (mumble_web.build_base | default(mumble_web_build_base)) | regex_replace('\\/$', '') }}"
+    deploy_path: "{{ (mumble_web.deploy_path | default(mumble_web_deploy_path)) | regex_replace('\\/$', '') }}"
+    version: "{{ mumble_web.version }}"
+    checksum: "{{ mumble_web.checksum }}"
+    reinstall: "{{ mumble_web.reinstall | default(false)}}"
+    config: "{{ mumble_web.config | default({}) }}"
+
+- name: Calculate facts for installation
+  set_fact:
+    archive_file: "{{ global_cache_dir }}/mumble-web-{{ version }}.tar.gz"
+    build_dir: "{{ build_base }}/mumble-web-{{ version }}"
+    deploy_base: "{{ deploy_path | dirname }}"
+
+- name: Create deployment parent directory
+  file:
+    path: "{{ deploy_base }}"
+    state: directory
+
+- name: Calculate facts for checking if install is neccesarry
+  set_fact:
+    deployed_version_file: "{{ deploy_base }}/.{{ deploy_path | basename }}_ansible_deployed_version"
+
+- name: Make sure deployed version file exists
+  file:
+    path: "{{deployed_version_file}}"
```

```suggestion
    path: "{{ deployed_version_file }}"
```
Please add spaces after/before curly brackets

---

ack